src/frontend/context/StegContext.ts

```diff
@@ -20,9 +20,9 @@ interface Props {
 
 export const [StegProvider, useSteg] = constate(({ fane, behandling }: Props) => {
     const erStegRedigerbar = fane && behandling.steg === faneTilSteg[fane];
-    const behandlingOgStegErRedigerbar = erStegRedigerbar && erBehandlingRedigerbar(behandling);
+    const stegErRedigerbar = erStegRedigerbar && erBehandlingRedigerbar(behandling);
 
     return {
-        behandlingOgStegErRedigerbar,
+        stegErRedigerbar,
```

Comment:
Kunde vi brukt `erStegRedigerbar`? 
Vanlig naming convention for booleans er `is`, `has` etc som prefix 👀

Reply:
Ja, vet det er konvensjonen men kanskje mindre lettlest? 🤔  Ref https://github.com/navikt/tilleggsstonader-sak-frontend/pull/234#discussion_r1525986180

Begge deler funker for meg

Reply:
Merger, men endrer navn siden hvis behov.